modin/api/partition_api.py

```diff
@@ -78,3 +81,110 @@ def _unwrap_partitions(oid):
             part.coalesce(bind_ip=bind_ip).unwrap(squeeze=True, bind_ip=bind_ip)
             for part in partitions
         ]
+
+
+def create_df_from_partitions(partitions, axis, engine):
+    """
+    Create DataFrame from remote partitions.
+
+    Parameters
+    ----------
+    partitions : list
+        List of Ray.ObjectRef/Dask.Future referencing to partitions in depend of the engine used.
+        Or list containing tuples of Ray.ObjectRef/Dask.Future referencing to ip addresses of partitions
+        and partitions itself in depend of the engine used.
+    axis : None, 0 or 1
+        The `axis` parameter is used to identify what is the partitions passed.
+        You have to set:
+        - `axis` to 0 if you want to create DataFrame from row partitions.
+        - `axis` to 1 if you want to create DataFrame from column partitions.
+        - `axis` to None if you want to create DataFrame from 2D list of partitions.
+    engine : int
+        The parameter is used to choose the engine for remote partitions which DataFrame will be created from.
+        Possible variants: ("PandasOnRayFramePartition", "PandasOnDaskFramePartition").
+        You have to specify index of the set: 0 - PandasOnRayFramePartition, 1 - PandasOnDaskFramePartition.
+
```

Comment:
why do we prefer to use engine index instead of its names or its FactoryObjects, from which, [it seems](https://github.com/modin-project/modin/blob/d720579bfa35bc322f11907c4511eb3ccf57fbd9/modin/data_management/factories/factories.py#L26-L29), we can get the whole required information about the engine.

Reply:
I want to minimize user's efforts to call the function. If we require to pass string names, letters amount will be increased. If we require to pass FactoryObjects, users will have to do addition import. However, in order to avoid extra imports in the function body we can use factories. I am doing respective changes. Any objections?

Reply:
Do not reduce writing complexity. Always strive to reduce reading and understanding complexity, as we write code about once, but read quite a few times (and usually by several people) :) so optimize for what has more prioirity ;)

Reply:
why do we ever need to specify a factory with a function parameter? AFAIK Modin does not provide a way to change the execution engine on the fly, so the factory determined only once when the user imports Modin, any example when we need an ability to specify engine?

Moreover, going by doc-string we obtain a list of Dask or Ray partitions, isn't a type of partitions determine which factory we **must** to use? If it is, we can get that information from partition types without an argument about factory type.

Reply:
You can change the engine on the fly (and the backend, for that matter), that's how cloud feature works. :)
If a code is dependent in its path about engine it should subscribe to its changes, etc.

Reply:
Removed `factory` parameter